docs/platform/nodebalancer/force-all-connections-on-ssl-with-nodebalancers.md

```diff
@@ -0,0 +1,352 @@
+
+3.  Copy your passphraseless private key into the **Private Key** field.
+
+4.  On your NodeBalancer `Configurations` page, select `Create Configuration`, you will need to create one for each port/protocol that you would like to use, i.e. `80` and `443`.
+
+5.  Under `Edit Configuration`  Once selected, fill out the values in the fields as shown below:
+ 
+         **Port**                    443
+         **Protocol**                HTTPS
+         **Algorithm**               Round Robin
+         **Session Stickiness**      None
+         **Certificate**             < Insert your signed SSL Certificate >
+         **Private Key**             < Insert your Private Key >
+         **Health Check Type**       HTTP Valid Status
+         **Check Interval**          5
+         **Check Timeout**           3
+         **Check Attempts**          2
+         **Check HTTP Path**         /
+
+    Then, select **`Save Changes`**. 
+
+6.  Add as many nodes as you require for the port configuration by selecting **`Add Node`**. Once selected, fill out the values in the fields like so:
+
+         **Label**                   < Backend Linode 1 >
+         **Address**                 < xxx.xxx.xxx.xxx:80 >
+         **Weight**                  100
+         **Mode**                    Accept
+
+    {: .note }
+    > 
+    > In the section labelled **`Address`**,  you will need to replace `xxx.xxx.xxx.xxx` by specify the private IP address that has been provisioned for that particular Linode, followed by a colon and `80`. This is because SSL is terminating at the NodeBalancer.
+
+    Then, select **`Save Changes`**.
+
+
+## Configuring your Web Server with a 301 Redirect
+
+### Configuring the Apache Webserver.
+
+1.  Enable mod_rewrite so that you can redirect all traffic back to the NodeBalancer over port 443/HTTPS. Enter the following command:
+
+        a2enmod rewrite
+     
+    or, you can load the module manually by appending the following to your Apache configuration file:
+
+        LoadModule rewrite_module modules/mod_rewrite.so
+
+    {:.note}
+    > Depending on if you are using a Debian or a Redhat based distribution, this file will be located in one of the following locations:
+    >
+    >     /etc/apache2/apache2.conf
+    >
+    >     /etc/httpd/httpd.conf
+
+2.  Now edit the Apache vhost configuration file to establish the rewrite rules necessary in order to redirect all incoming traffic from port 80/HTTP, back to the NodeBalancer on port 443/HTTPS:
+
+    {: .file-excerpt }
+    /etc/apache2/sites-available/example.com.conf
+    :   ~~~ apache
+        <VirtualHost *:80>
+          
+        ...
+          
+             RewriteEngine    On
+             RewriteCond      %{HTTP:X-Forwarded-Proto} !https
+             RewriteRule      ^.*$ https://%{SERVER_NAME}%{REQUEST_URI} [L,R=301,NE]
+             RewriteLog       /var/log/apache2/rewrite.log
+             RewriteLogLevel  5
```

Comment:
Lines 294/295 are deprecated. RewriteLog and RewriteLogLevel are deprecated in Apache 2.4 (still good for Apache2.2).